bpf/mock/helpers.sed

```diff
@@ -0,0 +1 @@
+1 s/^/\/\* GENERATED FROM lib\/helpers\.h \*\/\n/; /#ifndef BPF_/,/#endif/d; /#if __ctx_is/,/#endif/d; s/"compiler\.h"/<bpf\/compiler\.h>/g; /ctx\/ctx\.h/d; s/__BPF_HELPERS\(.*__\)/__MOCK_HELPERS\1/g; s/static //g; s/BPF_\w*(\(\w*\), /\1(/g; s/BPF_\w*(\(\w*\)/\1(/g; s/__printf(.*) //g; / =/{ N; s/ =.*;/;/ };
```

Comment:
Can you add copyright info and comment that describes the operation?

Reply:
Sorry, forgot it.